frontend/src/components/Pages/YTC/Executor/TokenResult.tsx

```diff
@@ -0,0 +1,100 @@
+import { Flex } from "@chakra-ui/react";
+import { shortenNumber } from "../../../../utils/shortenNumber";
+import { BaseTokenPriceTag, YTPriceTag } from "../../../Prices";
+
+interface TokenResultProps {
+    token: {
+        name: string,
+        amount: number,
+    };
+    baseTokenName?: string,
+    tokenType: "YToken" | "BaseToken";
+    trancheAddress: string,
+}
+
+export const TokenResult: React.FC<TokenResultProps> = (props) => {
+    const { token, tokenType } = props;
+
+    let tokenNameElement;
+    const tokenName = token.name.split('-')
```

Comment:
is this when the token names are like PTs with a data attached to them?

Reply:
if so, can you add a comment on when this is triggered?

Reply:
Yeah that was the original intention, it doesn't actually function as desired though. I'll add a comment to explain.

Reply:
I ended up fixing this functionality and adding some inline comments to explain it